src/AutoMapper/AbstractAutoMapper.php

```diff
@@ -88,7 +88,7 @@ public function map($value, string $target, Context $context = null)
 
     public function getConfiguration(string $source, string $target): ?MapperConfigurationInterface
     {
-        if (!array_key_exists($source, $this->configurations) || !array_key_exists($target, $this->configurations[$source])) {
+        if (!\array_key_exists($source, $this->configurations) || !\array_key_exists($target, $this->configurations[$source])) {
```

Comment:
Is this `\` syntax used on all source code?

If not, I think it should be reverted due to consistency.

Reply:
This is actually not in `php-cs-fixer` configuration but since it's considered as a "good practise" I see no point on removing it.